app/src/test/java/com/aws/amazonlocation/TestUtil.kt

```diff
@@ -7,11 +7,11 @@ import android.net.NetworkCapabilities
 import androidx.test.core.app.ApplicationProvider
 import com.aws.amazonlocation.mock.END_DATE_NULL
 import com.aws.amazonlocation.mock.START_DATE_NULL
+import java.text.SimpleDateFormat
+import java.util.*
```

Comment:
Lets avoid using wildcard imports. Please specify the exact list of classes that need to be imported.